.github/workflows/ci-delta-app.yml

```diff
@@ -0,0 +1,42 @@
+name: Delta App
+on:
+  pull_request:
+    paths:
+      - 'delta/**'
```

Comment:
Is this correct? Shouldn't it be `delta/app/**`?

Reply:
No, this should be triggered on all changes, as it's dependent on all parts of the codebase